dev/com.ibm.ws.kernel.feature.cmdline/src/com/ibm/ws/kernel/feature/internal/generator/FeatureList.java

```diff
@@ -368,7 +368,13 @@ private List<Map<String, Object>> getJavaVersion(ManifestFileProcessor mfp, Prov
 
             for (FeatureResource res : fd.getConstituents(SubsystemContentType.BUNDLE_TYPE)) {
                 ContentBasedLocalBundleRepository repo = mfp.getBundleRepository(fd.getBundleRepositoryType(), null);
-
+                
+                if(res.getJavaRange() != null) {
+                	// Bundles & Jars with the java range attribute should not be considered for the supported java versions 
```

Comment:
Technically, it's not the "java range attribute", but require-java. 

Bundles that use require-java prior to my java range should still have the correct documentation generated, I hope?